docs/builders/webpack.md

```diff
@@ -0,0 +1,201 @@
+---
+title: 'Webpack'
+---
+
+Storybook displays your components in a custom web application built using [Webpack](https://webpack.js.org/). Webpack is a complex tool, but our default configuration is intended to cover most use cases. [Addons](https://storybook.js.org/addons/) are also available that extend the configuration for other everyday use cases.
+
+You can customize Storybook's webpack setup by providing a `webpackFinal` field in [`.storybook/main.js`](../configure/overview.md#configure-your-storybook-project) file.
+
+The value should be an async function that receives a webpack config and eventually returns a webpack config.
+
+### Default configuration
+
+By default, Storybook's webpack configuration will allow you to:
+
+#### Import images and other static files
+
+You can import images and other local files and have them built into the Storybook:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/my-component-story-import-static-asset.js.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+#### Import JSON as JavaScript
+
+You can import `.json` files and have them expanded to a JavaScript object:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/my-component-story-import-json.js.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+If you want to know the exact details of the webpack config, the best way is to run either of the following:
+
+```shell
+
+## Development mode
+yarn start-storybook --debug-webpack
+
+## Production mode
+yarn build-storybook --debug-webpack
+```
+
+### Bundle splitting
```

Comment:
Should we just rename this to code splitting? That's the more common name and we refer to it as code splitting everywhere else.

Reply:
yea!